scripts/import/import_EPMC.pl

```diff
@@ -97,7 +96,7 @@
     import_citations($reg, $file_data, $type);
 
     ## update variations & variation_features to have Cited status
-    update_evidence($dba) unless defined $no_evidence;
+    ## update_evidence($dba) unless defined $no_evidence;
```

Comment:
It would be better to remove all of these update  evidence lines than hash them out.